tests/integration/templatetags/test_currency_filters.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from decimal import Decimal as D
 
+from django.utils import translation
 from django.test import TestCase
 from django import template
```

Comment:
Yeah I realise that, but we have to try to keep the source neat. Letting this slip sometimes will add-up in the long run :(